src/mainWindow.js

```diff
@@ -41,12 +41,12 @@ setInterval(() => {
 
   host.insertAdjacentElement('afterend', oaVersion);
 
-  let advanced = document.querySelector('[class*="socialLinks"]').parentElement.querySelector('[class*="premiumTab"] ~ [class*="header"] + [class*="item"] + [class*="item"] + [class*="item"] + [class*="item"] + [class*="item"] + [class*="item"] + [class*="item"] + [class*="item"] + [class*="item"] ~ [class*="separator"]')?.previousSibling;
-  if (!advanced) advanced = [...document.querySelectorAll('[class*="item"]')].find(x => x.textContent === 'Advanced');
+  let advanced = document.querySelector('[class^=side] :nth-last-child(1 of [class^=header]:not([class^=headerText])')?.previousElementSibling?.previousElementSibling;;
   if (!advanced || document.getElementById('openasar-item')) return;
 
   const oaSetting = advanced.cloneNode(true);
   oaSetting.textContent = 'OpenAsar';
+  oaSetting.setAttribute('aria-label', 'OpenAsar');
```

Comment:
Unnecessary as it is just a duplicate of text content

Reply:
it's a duplicate, however if we don't do that, then an advanced tag is being left there
imo, either it should be changed to this or we could just remove the attribute as some have the attributes and some don't